src/foam/core/AbstractFObject.js

```diff
@@ -91,6 +91,15 @@ foam.CLASS({
     }
   ],
 
+  properties: [
```

Comment:
Does this work? Does this property get included on the JS side? Shouldn't you add it to FObject.js instead?

Reply:
Tried it with both and also only on FObject.js, just adding the property doesn't work for some reason the console throws errors saying the setters and getters aren't found. We can talk about it more on Monday.

Reply:
Add to FObject and include a javaType, otherwise it won't get generated on the Java side. You might have to add it as a refinement to FObject since type support may not yet be present when FObject is first loaded.